### ISSUE TITLE
E-Mail image sizing wrong in Outlook

### ISSUE THREAD

**Issue by danir-de** [nextcloud/server#32790]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Images like the logo are not displayed in the correct size when using the Outlook Desktop (all versions from 2013 to the latest 2021 are affected).

This results in you having to downscale the logo-image of your instance, in order for it not to fill the whole screen when opening mails. You can't upload a image to small, since it would look pixelated on some places in the GUI otherwise, so the logo is always too big in any email.

I suppose that many organizations are affected by this, since Outlook is a widespread virus.

### Steps to reproduce

1. Upload a high resolution logo.
2. Trigger anything that sends an email.
3. Open the email with any Outlook client.
4. Look at the unresponsive monstrosity of a logo:
![image](https://user-images.githubusercontent.com/4672840/172870888-308f0b44-cd69-402e-9970-6fdec7aed5e9.png)

### Expected behavior

I would expect it to look sleek and elegant exactly on any other modern mail client or webmail client:
![image](https://user-images.githubusercontent.com/4672840/172871220-6afd9c22-9ef3-4fb7-8d17-fd3258c2f5be.png)

### Installation method

Manual installation

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.0

### Web server

Nginx

### Database engine version

MariaDB

**Comment by andyxheli** [comment]
Same issue.

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

**Comment by andyxheli** [comment]
Same issue on NC 25.0.3

**Comment by 0rb1ta** [comment]
I confirm, I have the same problem, Outlook ignores styles like  min-width:;max-height:; padding:;border-radius:; max-height:;max-width

```
<table class="row collapse" style="border-collapse:collapse;border-spacing:0;display:table;padding:0;position:relative;text-align:left;vertical-align:top;width:100%">
<tbody>
<tr style="padding:0;text-align:left;vertical-align:top">
<center data-parsed="" style="background-color:#0082c9;**min-width:175px;max-height:175px; padding:35px 0px;border-radius:200px**">
<img class="logo float-center" src="https://blablabla.com/apps/theming/image/logo?useSvg=0,&,v=35" alt="BLABLABLA" align="center" style="-ms-interpolation-mode:bicubic;clear:both;display:block;float:none;margin:0 auto;outline:0;text-align:center;text-decoration:none;**max-height:105px;max-width:105px**;width:auto;height:auto">
</center>
</tr>
</tbody>
</table>
```

**Comment by DeadPo01** [comment]
NextCloud 25.0.4 (on TrueNas 13.0.4)
Outlook 2016 can't even view logo image....
And google chrome not opening logo, but offers to download it, because logo goes without extension.
The link in message looks like:
` src="https://domain_name/index.php/apps/theming/image/logo?useSvg=0&amp;v=10"`

Image in outlook looks like:

![error](https://user-images.githubusercontent.com/70272456/225535072-8a04394c-b251-431b-bfe2-324c51624ccc.png)

**Comment by DeadPo01** [comment]
No. I changed logo and tested email AFTER that. Tested again now (logo changed about a month ago) and.... no logo in message.
If i insert link from message image into browser it offers me to download file "logo" without extension. After download if add .svg extension to this file - image will open.
No. I changed logo and tested email AFTER that. Tested again now (logo changed about a month ago) and.... no logo in message.
If i insert link from message image into browser it offers me to download file "logo" without extension. After download if add .svg extension to this file - image will open.
No. I changed the logo and tested email AFTER that. Tested again now (logo changed about a month ago) and.... no logo in the message.
If I insert links from message image into a browser, it offers me to download file "logo" without extension. After download if add an .svg extension to this file - image will open.
